src/react-app-env.d.ts

```diff
@@ -1 +1,9 @@
 /// <reference types="react-scripts" />
+
+import { MetaMaskProvider } from "./domain";
+
+declare global {
+  interface Window {
+    ethereum?: MetaMaskProvider;
```

Comment:
This should be an `ExternalProvider` (and ideally, `unknown`).

Reply:
Moved to a `ExternalProvider` by the moment